spec/components/schemas/content/optin_request.ts

```diff
@@ -0,0 +1,62 @@
+import { SchemaObject } from 'openapi3-ts';
+import { ref as baseRef } from './base';
+import { createComponentRef } from '../../../../utils/ref';
+
+const notificationOptIn: SchemaObject = {
+  type: 'object',
+  allOf: [{
+    $ref: baseRef,
+  }, {
+    type: 'object',
+    properties: {
+      type: {
+        type: 'string',
+        example: 'optin_request',
+      },
+      medialUrl: {
+        description: 'URL for the thumbnail the user will see.',
+        type: 'string',
+        example: 'https://cdn.zenvia.com/main-picture.jpg',
+      },
+      title: {
+        description: 'The topic of the recurring notification subscription.',
+        type: 'string',
+        example: 'Weekly Sale'
+      },
+      frequency: {
+        description: 'How often the messages will be sent after user opted-in.',
+        type: 'string',
+        enum: [
+          'DAILY',
+          'WEEKLY',
+          'MONTHLY'
+        ]
+      },
+      reOptIn: {
+        description: 'Allow user to opt-in back again after opting-out.',
+        type: 'string',
+        enum: [
+          'ENABLED',
+          'DISABLED'
+        ],
+        default: 'ENABLED',
+      },
+      payload: {
+        description: 'Content to be sent back as a message when the user clicks the button. By default, the `title` of the request will be used as payload.',
+        type: 'string',
+      },
+      timezone: {
+        description: `Indicates the timezone that will be used for scheduling the notifications.<br>
```

Comment:
Por último, priorizando o conceito de recurring messages, que tal:
`Indicates the timezone that will be used for scheduling the recurring messages.`

Reply:
Concordo. Acho muito melhor.
Eu tinha pensado que iriamos usar **notification** em tudo :smile:

Reply:
Sim, mas conforme fui vendo o que você fez e o que fui sugerindo, acho que vai ficar melhor sem o notification, já que a gente usa message pra tudo, e mesmo as notificações de whatsapp não chamos de notificação e sim de template por conta do conteúdo.